### PR TITLE
fix a mem-leak

### PR DESCRIPTION
**Pull request by chenhuitao**
fix oss-fuzz issue 10334: unicorn/fuzz_emu_x86_32: Direct-leak in g_malloc.

### CODE REVIEW

**Comment by aquynh** [comment]
this breaks both Travis & Appveyor, can you take a look?

**Comment by Waterman178** [comment]
thanks

File: qemu/translate-all.c [hunk 1]
```diff
@@ -135,24 +135,55 @@ static void cpu_gen_init(struct uc_struct *uc)
...
+    int i, x;
+    void **p;
+
+    if (uc) {
+        if (uc->l1_map) {
+            x = V_L1_SHIFT / V_L2_BITS;
+            if (x <= 1) {
+                for (i = 0; i < V_L1_SIZE; i++) {
+                    p = uc->l1_map[i];
+                    g_free(p);
```
Comment (aquynh) [inline qemu/translate-all.c]: do we need to check if p is NULL here? 
(at lest for consistency, because the code above does that)

Comment (chenhuitao) [reply qemu/translate-all.c]: Not be necessary,  g_free() accepts NULL pointer. But you are right, it should be checked. I will add it.

**Comment by aquynh** [comment]
merged, thanks!

**Pull request merged**